### PR TITLE
Buffs Firelockers + Adds New Tool: The Rescue Axe 

### PR DESCRIPTION
**Pull request by WickedCybs**
I've ported the emergency forcing tool from Baystation, though here it has been renamed to the "rescue axe" as I feel that might be a more innately understandable name. It is a hand axe intended to be used to either pry open stuff or smash your way in via weak points like windows (which it deals its full damage to rather than just half). You can think of it as a crowbar with an axehand and pick at the top of it. 

It's supposed to be a piece of kit mainly used by engineering and those dealing with atmospheric issues like fires, so it has been put into the CE and atmos tech lockers along with fire closets. The engineering lockers also have the clear faced gas mask variants replacing the old classic full gas masks.

For the fire closets I've done a few things. Standard variants will have a small set of inflatables (one door, two walls) and "filled" variants have a slightly bigger set of inflatables (two doors, three walls). Medical's fire closet guarantees a ladder. The re-arrangement of the fire closets also had me take the chance to replace those old "classic" full gas masks in ss13 with the alternate version that has a clear view of the face. It fits the style of a firefighter far more I'd say.

image

### CODE REVIEW

**Comment by WickedCybs** [comment]
!review

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Comment by SleepyGemmy** [comment]
also, would it be possible to have the axe be red instead of blue? i can perform the necessary sprite changes in that case.

**Comment by WickedCybs** [comment]
!wip

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 1]
```diff
@@ -23,7 +23,9 @@
 	new /obj/item/storage/toolbox/mechanical(src)
 	new /obj/item/clothing/suit/storage/hazardvest/ce(src)
 	new /obj/item/clothing/mask/gas(src)
```
[the remark this answers is not in the document]
Comment (WickedCybs) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm:25]: Will consider it. Not entirely sure if I'll get anyone angry I took their masks away.

**Comment by WickedCybs** [comment]
> also, would it be possible to have the axe be red instead of blue? i can perform the necessary sprite changes in that case.

Was discussed on discord but will say my answer here again. That's fie.

**Comment by WickedCybs** [comment]
Made the PR a WIP for now, because when I get back to it the FR's need their closet changed so no rescue axe spawns in it. Other spawn points might need to be looked at too. That said, this is to be an uncommon item at best, bit rare at worst. I feel like overall players can be trusted with it and there's more than enough things around that do nearly the same or more than the amount of damage listed on the axe (12 force) which are publicly accessible to the crew.

Another thing is, I want to look into the tool doing double damage to doors too.

**Review by NonQueueingMatt — APPROVED** [review]
Not really sure about this being in every single fire locker, but I guess we can give it a try?

**Review by JohnWildkins — APPROVED** [review]

**Comment by WickedCybs** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by Arrow768** [comment]
I am sceptical about this, as it makes it extremely easy to smash down windows/grilles.
But I guess we´ll see how it goes.

**Pull request merged**